script/Ciel/Day 3/03_01_CIEL03_3_3B.txt

```diff
@@ -532,54 +573,63 @@ I'm not being nice to you for your sake, so please don't worry too much about it
 [sha:d9a0c1dbd0a91b21be9fe1c9e00f1f67ede5b779]{
 -- Page 232, Offset 5444.
 -- 「俺のためじゃ、ない……？」
-"It's not for me?"
+"You aren't doing it for my sake...?"
 }
 [sha:4f227e9d57eaddb973a26f3ab8fbe17e992ab502]{
 -- Page 233, Offset 5445. Mods: @n.
 -- 「はい。
 "Yes.
 }
-[sha:296153b5ac049cb48510eacb5052856468765762]{
+[offset:5446]{
 -- Page 233, Offset 5446. Glued. Mods: @k, @e.
 -- ……
-...
+%{n}
 }
 [sha:eeb54dbd0d2eb2b2199873edb38a0cfdb17f8abc]{
 -- Page 233, Offset 5447. Glued. Mods: @x.
 -- その、ですね。わたしは、わたしが遠野くんに優しくしてあげたいから、こうしているんです。要するにわたしの趣味です。遠野くんの事情はあまり関係ありません。
-You see... I'm doing this because I want to be nice to you. In short, it's my hobby. Your situation doesn't really matter.
+// TLing 事情 as echoing back to 都合
+How to explain...
+ I'm doing this because I want to be nice to you.
+ Put simply, it's my hobby.
+ Your own reasoning doesn't really enter into it.
 }
 [sha:9c513106547585776ea91b23d5f859638ead0c0c]{
 -- Page 233, Offset 5448.
 -- 仮に貴方が悪人でもわたしは手を差し伸べます。まあ、そんなコトは百歩譲ってもないでしょうけど」
-// B: Arcueid is a fucking vampire and she's not the most unrealstic heroine :aokoheh:
-Even if you had become a bad person, I'd reach out to you. Well, that sort of thing couldn't happen outside of a hypothetical."
+// B: Arcueid is a fucking vampire and she's not the most unrealistic heroine :aokoheh:
+Even if you %{i}had%{/i} become a bad person, I'd still reach out to you.
+ Well, not that I think you ever actually would."
 }
 [sha:806e98a6936eca1dab7243ad1568d410bb8134d9]{
 -- Page 234, Offset 5449.
 -- 「あの、趣味って……先輩、人助けが趣味なんですか？」
-"What do you mean by 'hobby'... Senpai, is helping people a hobby?"
+"Hold on, 'hobby'...?
+ Senpai, are you saying it's your hobby to help people?"
 }
 [sha:2f0c6f97fbab4f4d34b2cd372121850c598aaa3e]{
 -- Page 234, Offset 5450.
 -- あまりの断言っぷりに、つい軽口がこぼれてしまった。
-She was so assertive that I couldn't help but let out a little chuckle.
+I'm so stunned by her declaration that I can't help teasing her a little.
 }
 [sha:63ae1cb3729c2a02e7cc5861b0979c7786d851c3]{
 -- Page 234, Offset 5451.
 -- 先輩は気分を害さず、逆に、我が意を得たりとばかりに胸を張った。
-Senpai wasn't offended. On the contrary, she seemed proud, as if she understood what I'd meant by it.
+Senpai doesn't seem to be offended.
+ On the contrary, she puffs up with pride, as though I've hit the nail right on the head.
```

Comment:
I'd hit

Reply:
> as though I have hit the nail on the head
> as though I had hit the nail on the head

`I have` feels _slightly better to me because it's still perfect aspect for hitting the nail but more... recent? Like he's still very close to the state of hitting (happened last sentence) whereas 'had' makes it sound like something he did a while ago.

Reply:
I guess I can see what you're getting at but I don't really agree to how its applied here. I still think "had" sounds right